python/lsst/sims/maf/plots/plotHandler.py

```diff
@@ -9,6 +9,21 @@
 
 __all__ = ['PlotHandler', 'BasePlotter']
 
+
+def none_min(x):
+    if None in x:
```

Comment:
Is it ok if I push something to this later today? (On phone, not sure if it's already sorted)